src/testdrive/parser.rs

```diff
@@ -107,10 +119,14 @@ fn parse_builtin(line_reader: &mut LineReader) -> Result<BuiltinCommand, InputEr
 
 fn parse_sql(line_reader: &mut LineReader) -> Result<SqlCommand, InputError> {
     let (_, line1) = line_reader.next().unwrap();
+    let query = line1[1..].trim().to_owned();
     let line2 = slurp_one(line_reader);
     let line3 = slurp_one(line_reader);
     let mut column_names = Vec::new();
     let mut expected_rows = Vec::new();
+    lazy_static! {
+        static ref HASH_REGEX: Regex = Regex::new(r"(\S+) values hashing to (\S+)").unwrap();
```

Comment:
Can you anchor this regex `^(\S+) values hashing to (\S+)$` so that it doesn't match something in quotes? (Just in case someone one day wants to test for the literal string "5 values hashing to blah".)